src/include/duckdb/execution/operator/csv_scanner/csv_file_scanner.hpp

```diff
@@ -110,5 +113,10 @@ class CSVFileScan {
 	CSVReaderOptions options;
 
 	CSVIterator start_iterator;
+
+private:
+	vector<string> names;
```

Comment:
doe we really need both names/types AND the columns here?

Reply:
I first removed types+names in favor of just columns, but that cascaded into 5-6 other file changes